terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionLevelResolverFilter.java

```diff
@@ -32,6 +32,9 @@ protected void doFilterInternal(HttpServletRequest request,
             HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
 
         String path = request.getServletPath();
+        if (path == null || path.length() == 0) {

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...